### PR TITLE
Travis setup

### PR DESCRIPTION
**Pull request by laythq**

### CODE REVIEW

**Comment by elishkaflint** [comment]
:ant:

File: package.json [hunk 1]
```diff
@@ -3,15 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "istanbul": "^0.4.5",
```
Comment (bibicollins) [inline package.json:6]: Does code climate need istanbul?

Comment (laythq) [reply package.json:6]: duh

Comment (bibicollins) [reply package.json:6]: But we are using inbuilt jest coverage

**Review by bibicollins — APPROVED** [review]
:ant:

**Review by elishkaflint — APPROVED** [review]

**Pull request merged**